wiki/Tournaments/TWC/2015/de.md

```diff
@@ -2,155 +2,526 @@
-3. Jeder Mappool besteht aus 19 Maps.
-4. Jeder Mappool hat eine Tiebreaker Map.
-5. Die NoMod Gruppe wird ohne aktiven Mods gespielt.
-6. Die Hidden, HardRock und DoubleTime Gruppe wird mit dem entsprechenden, aktivierten Mod gespielt.
-7. Die FreeMod Gruppe schaltet den FreeMod frei. Jeder einzelne Spieler kann dann Hidden, HardRock, Flashlight oder mit gar keinen Mod spielen.
-   1. Spieler dürfen mehr als einen Mod auswählen.
-   2. Wenn eine FreeMod Map gespielt wird, müssen mindestens 1 Spieler jedes Teams mindestens einen Mod aktiviert haben.
-8. Die Tiebreaker Map wird mit denselben Bedingungen wie beim FreeMod gespielt.
-   1. Wenn die Tiebreaker Map gespielt wird, muss jedoch niemand einen Mod aktiviert haben.
-9. Die Größe der NoMod Gruppe besteht aus 6 Maps in allen Etappen.
-10. Die Größe der mod-spezifischen Gruppen besteht aus 3 Maps in allen Etappen.
+1. Es wird einen Mappool für die Gruppenphase, einen Mappool für das Achtelfinale, einen Mappool für das Viertelfinale, einen Mappool für Halbfinale und einen Mappool für das Finale geben (Woche 1 & Woche 2 verwenden denselben Mappool).
+2. Jeder Mappool besteht aus 5 Klassen: NoMod, HardRock, Hidden, DoubleTime und FreeMod
+3. Jeder Mappool besteht aus insgesamt 19 Maps.
+4. Jeder Mappool beinhaltet einen Tiebreaker.
+5. Der NoMod-Stapel wird ohne aktive Mods gespielt.
+6. Die Hidden-, HardRock- und DoubleTime-Stapel werden mit der entsprechenden, aktivierten Mod gespielt.
+7. Der FreeMod-Stapel schaltet den FreeMod frei. Jeder einzelne Spieler kann dann mit Hidden, HardRock, Flashlight oder gar keinen Mods spielen.
+   - Wenn eine FreeMod-Map gespielt wird, muss mindestens 1 Spieler jedes Teams eine Mod aktiviert haben.
+8. Die Tiebreaker-Map wird mit denselben Bedingungen wie beim FreeMod gespielt.
+   - Wenn die Tiebreaker-Map gespielt wird, muss niemand einen Mod aktiviert haben.
+9. Die Größe des NoMod-Stapels besteht in allen Etappen aus 6 Maps.
+10. Die Größe der mod-spezifischen Stapel besteht in allen Etappen aus 3 Maps.
 
 ### Zeitplananweisungen
 
-1. Jede Etappe wird **einzeln am Wochenende** abgehalten.
+1. Jede Etappe wird **an einem einzelnen Wochenende** abgehalten.
 2. Partien können sich in der Gruppenphase zeitlich überlappen.
-3. Alle Double-Elimination Etappen finden entweder am Samstag oder am Sonntag statt.
-4. Die Zeiten der Partien werden vom Turnier Management festgelegt. Die Termine werden am Sonntag angekündigt, bevor die eigentliche Etappe stattfindet. Der Turnier Management bemüht sich die Zeiten so gut wie möglich an den Zeitzonen der teilnehmenden Spieler anzupassen.
-5. Die Kapitäne sind für die Anwesenheit deren Teams zuständig. Eine größere Teamgröße stellt sicher, dass jedes Team mindestens 4 Spieler in jeder Partie zur Verfügung hat. Es müssen vom jeweiligen Team mindestens 3 Spieler anwesend sein, ansonsten zählt die Partie als aufgegeben.
+3. Alle Double-Elimination-Etappen finden entweder am Samstag oder am Sonntag statt.
```

Comment:
As mentioned above, I will let you decide to choose either `Doppeleleminierungsetappen` or the other here.